src/auth/oauth/test/oauth.test.ts

```diff
@@ -25,6 +24,31 @@ beforeEach(() => {
   (Cookies as any).mockClear();
 });
 
+declare global {
+  interface Matchers<R> {
+    toBeWithinSecondsOf(compareDate: number, seconds: number): R;
+  }
+}
+
+expect.extend({
```

Comment:
I'd suggest moving this to the test helper since it's pretty useful throughout all tests!